actions/action_set_package_url.py

```diff
@@ -0,0 +1,32 @@
+from typing import Dict, Text, Any, List
+
+from rasa_sdk import Tracker
+from rasa_sdk import Action
+from rasa_sdk.executor import CollectingDispatcher
+from rasa_sdk.events import SlotSet
+
+
+class ActionSetPackageUrl(Action):
+    def name(self) -> Text:
+        return "action_set_package_url"
+
+    def run(
+        self,
+        dispatcher: CollectingDispatcher,
+        tracker: Tracker,
+        domain: Dict[Text, Any],
+    ) -> List[Dict[Text, Any]]:
+
+        province = tracker.get_slot("province")
+        risk_level = (
+            "elevated-vocid-risk"
+            if tracker.get_slot("elevated-covid-risk")
+            else "common"
+        )
+
+        return [
+            SlotSet(
+                "package_url",
+                f"https://covid19.dialogue.co/#/info?id=ca-{province},{risk_level}",
+            )
+        ]
```

Comment:
I don't think we should store this in a slot, it could change in the future. I feel like this belongs in the message. I saw that we were calling utter_visit_package after this action. Maybe we could call dispatcher.utter_template("utter_visit_package") here and set province and risk_level variables in the message.